services/app-api/src/resolvers/indexHealthPlanPackages.ts

```diff
@@ -1,50 +1,62 @@
+import { Span } from '@opentelemetry/api'
 import { ForbiddenError } from 'apollo-server-lambda'
-import { isStateUser, HealthPlanPackageType } from '../domain-models'
+import { isStateUser, HealthPlanPackageType, isCMSUser } from '../domain-models'
 import { QueryResolvers } from '../gen/gqlServer'
 import { logError, logSuccess } from '../logger'
-import { isStoreError, Store } from '../postgres'
+import { isStoreError, Store, StoreError } from '../postgres'
 import {
     setErrorAttributesOnActiveSpan,
     setResolverDetailsOnActiveSpan,
     setSuccessAttributesOnActiveSpan,
 } from './attributeHelper'
 
+const validateAndReturnHealthPlanPackages = (
```

Comment:
Nice way to do this. I was imagining leaving this below and doing some kind of 
```
let results = undefined
if (isStateUser) {
  results = findAllByState()
} else if (isCMSUser) {
  results = findAll()
}

… do stuff with results
```
but your version doesn’t have to deal with the undefined `results` type which is nice.